ofono/src/gprs.c

```diff
@@ -854,9 +855,10 @@ static void pri_reset_context_settings(struct pri_context *ctx)
 
 	pri_context_signal_settings(ctx, signal_ipv4, signal_ipv6);
 
-	if (ctx->type == OFONO_GPRS_CONTEXT_TYPE_MMS) {
```

Comment:
Where is the matching closing } bracket? Why is this removed?

Reply:
This is coming from commit e52746aed, if you read it, the opening brace is added in a new
if clause a few lines under.

Reply:
Ah Im stupid yes